packages/tests/server/adapters/fetch.test.ts

```diff
@@ -266,3 +269,22 @@ test.each([
 
   await custom.close();
 });
+
+// https://github.com/trpc/trpc/issues/5659
+test('mutation', async () => {
+  const t = await startServer();
+  try {
+    const res = await Promise.all([
+      t.client.helloMutation.mutate('world'),
+      t.client.helloMutation.mutate('KATT'),
+    ]);
+    expect(res).toEqual(['hello world', 'hello KATT']);
+  } catch (err) {
+    console.log('err', err);
+    console.log('err', (err as any).cause);
+
+    throw err;
+  } finally {
+    await t.close();
+  }
+});
```

Comment:
No idea why my repro attempt last night didn't trigger the bug but this does? Would love to understand

Reply:
I'm not sure but in the snippet you shared you used a query and not a mutation, a query doesn't read the body